crates/forge/tests/cli/test_cmd.rs

```diff
@@ -1481,3 +1481,114 @@ Traces:
 ...
 "#]]);
 });
+
+forgetest_init!(gas_metering_reset, |prj, cmd| {
+    prj.wipe_contracts();
+    prj.insert_ds_test();
+    prj.insert_vm();
+    prj.clear();
```

Comment:
what is this for? the project starts out with forgestd already

Reply:
correct, thing is that the cheatcode is not yet available in forge-std therefore used `insert_vm` to import local Vm